docs/api/web-frame.md

```diff
@@ -44,7 +44,18 @@ Returns `Number` - The current zoom level.
 * `minimumLevel` Number
 * `maximumLevel` Number
 
-Sets the maximum and minimum zoom level.
+Sets the maximum and minimum pinch-to-zoom level. In future versions of Electron
+this method will be called `setVisualZoomLevelLimits`.
+
+### `webFrame.setLayoutZoomLevelLimits(minimumLevel, maximumLevel)`
+
+* `minimumLevel` Number
+* `maximumLevel` Number
+
+Sets the maximum and minimum layout-based (i.e. non-visual only) zoom level. In 
```

Comment:
Perhaps this change should be documented in https://github.com/electron/electron/blob/71f94c7a3ad1014a0c1fdca248046c4d61cb3988/docs/tutorial/planned-breaking-changes.md#L14